v1/mobileapps.js

```diff
@@ -37,6 +43,7 @@ class MobileApps {
             return this._fetchFromMCS(hyper, req);
         })
         .catch({ status: 404 }, () => this._fetchFromMCSAndStore(hyper, req))
+        .tap(this._injectDeprecationHeaders.bind(this))
```

Comment:
Is this symbol available in another patch? I think by convention `res.headers` is intended to fairly explicitly align with real HTTP header values, although there's nothing wrong with this `_injectDeprecationHeaders` value observing on that header and applying any other aspects it thinks are germane (e.g., for internal enrichment of headers above and beyond `Sunset:` or `Deprecation:` as it were).

Reply:
I am not sure if that's what you mean but here is the definition:
https://github.com/wikimedia/restbase/pull/1317/files#diff-6805c4ed202dfbf57927df5bdad305acce64cd6a04110293c3952017be860b82R22

Reply:
I glossed right over it, sorry!